merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/ProfileRepository.java

```diff
@@ -24,25 +25,33 @@ static ProfileSet getProfiles(
       final long datasetId,
       final Window simulationWindow
   ) throws SQLException {
-    final var realProfiles = new HashMap<String, Pair<ValueSchema, List<Pair<Duration, RealDynamics>>>>();
-    final var discreteProfiles = new HashMap<String, Pair<ValueSchema, List<Pair<Duration, SerializedValue>>>>();
+    final var realProfiles = new HashMap<String, Pair<ValueSchema, List<Pair<Duration, Optional<RealDynamics>>>>>();
+    final var discreteProfiles = new HashMap<String, Pair<ValueSchema, List<Pair<Duration, Optional<SerializedValue>>>>>();
 
     final var profileRecords = getProfileRecords(connection, datasetId);
     for (final var record : profileRecords) {
       switch (record.type().getLeft()) {
-        case "real" -> realProfiles.put(record.name(),
-                                        Pair.of(
-                                            record.type().getRight(),
-                                            getRealProfileSegments(connection, record.datasetId(), record.id(), simulationWindow)));
-        case "discrete" -> discreteProfiles.put(record.name(),
-                                                Pair.of(
-                                                    record.type().getRight(),
-                                                    getDiscreteProfileSegments(connection, record.datasetId(), record.id(), simulationWindow)));
+        case "real" -> realProfiles.put(
+          record.name(),
+          Pair.of(
+              record.type().getRight(),
+              getRealProfileSegments(connection, record.datasetId(), record.id(), simulationWindow)
+          )
+        );
+        case "discrete" -> discreteProfiles.put(
+            record.name(),
+            Pair.of(
+                record.type().getRight(),
+                getDiscreteProfileSegments(connection, record.datasetId(), record.id(), simulationWindow)
+            )
+        );
         default -> throw new Error("Unrecognized profile type");
       }
     }
 
-    return new ProfileSet(realProfiles, discreteProfiles);
+    final var result = new ProfileSet(realProfiles, discreteProfiles);
+    System.out.println(result);
```

Comment:
Maybe this was added for testing and was meant to be removed. If you'd like to keep it around it may help to give this message some context. :)

Reply:
ha, yup that's me debugging